### ISSUE TITLE
[Question] How to use checkpoint file to start training with a new learning rate?

### ISSUE THREAD

**Issue by muntasir2000** [wenet-e2e/wenet#538]
Hi,
I am training a U2++ conformer model with my own data. But I trained for 90+ epoch. It seems that the training and CV loss curve became steep. So, I want to restart the training with a new learning rate. How can I do that? I have tried editing the <epoch_number>.yaml and also the config.yaml but no avail. Thanks in advance.

**Comment by robin1001** [comment]
The checkpoint lr is automatically computed by `step` in checkpoint yaml and `lr` in your training config yaml. I think you can just decay the `lr` in your training config yaml.

**Comment by lyj157175** [comment]
Hello, I used aishell-1 to retrain the U2++conformer model. Why does the loss decrease slowly? The loss is basically around 60. And the model trained for 40 epochs is basically not recognized on the test set.

**Comment by robin1001** [comment]
> Hello, I used aishell-1 to retrain the U2++conformer model. Why does the loss decrease slowly? The loss is basically around 60. And the model trained for 40 epochs is basically not recognized on the test set.

What does retrain mean? You have the different problem which is not related to this issue, it's better to make a new PR.

**Comment by lyj157175** [comment]
ok
